spec/observables/dom/ajax-spec.ts

```diff
@@ -728,7 +716,6 @@ describe('ajax', () => {
     });
 
     it('should succeed on 204 No Content', () => {
-      const expected: null = null;
```

Comment:
`null` didn't make any sense here. a real `XHR` would never have `null` in the `responseText`, or the `response`.